api/controllers/Irc_EntryController.js

```diff
@@ -63,9 +64,9 @@ module.exports = {
   },
 
   process_event: function (request_body) {
-    if (request_body.operation === 'check in') {
+    if (eventNames.indexOf(request_body.operation) !== -1) {
       return this.saveDetainee(request_body)
```

Comment:
this won't work for updates.
you'll never actually update the `detainee` if the original one already exists